README.md

```diff
@@ -13,26 +13,10 @@ Before contributing code to Emperor, please familiarize yourself with the [contr
 
 ## Usage examples
 
-The main interface to create Emperor visualizations is the `make_emperor.py` script, inputing a mapping file and a PCoA data file, will generate an Emperor graphical user interface to analyze and visualize your data.
+We recommend using Emperor through its Python interface. The following, are two
+examples of how to do this:
 
-If you have a QIIME compliant mapping file and a PCoA file, try the following command from a terminal session:
+- [Visualizing data produced with scikit-bio](http://nbviewer.jupyter.org/github/biocore/emperor/blob/new-api/examples/keyboard.ipynb).
```

Comment:
❓ is that really the recommended way? Are we planning to have something similar to make_emperor.py? I like the idea of a script cause if I have a single pc/map-file and want to plot I don't need to have a full notebook.